src/rust/dbv2/migrations/2023-11-16-103707_market_volume/up.sql

```diff
@@ -0,0 +1,115 @@
+-- Your SQL goes here
+CREATE MATERIALIZED VIEW aggregator.markets_24h_data AS
+    WITH fills AS(
+        SELECT
+            *
+        FROM
+            fill_events
+        WHERE
+            "time" >= CURRENT_TIMESTAMP - '1 day'::interval
+            AND emit_address = maker_address
+    )
+    SELECT
+        market_id,
+        MIN(fills.price) AS min_price_24h,
+        MAX(fills.price) AS max_price_24h,
+        SUM(fills.size) AS base_volume_24h,
+        SUM(fills.size * fills.price) AS quote_volume_24h
+    FROM
+        fills
+    GROUP BY
+        market_id;
+
+
+DROP VIEW api.markets;
+
+
+CREATE VIEW api.markets AS
+    WITH max_time AS (
+        SELECT
+            MAX("time") AS mtime
+        FROM
+            fill_events
+    ),
+    last_fills AS(
+        SELECT
+            *
+        FROM
+            fill_events,
+            max_time
+        WHERE
+            mtime = "time"
+        AND
+            "time" >= CURRENT_TIMESTAMP - '1 day'::interval
+        ORDER BY
+            txn_version DESC,
+            event_idx DESC
+        LIMIT 1
+    ),
+    first_fills AS(
+        SELECT
+            *
+        FROM
+            fill_events
+        WHERE
+            "time" >= CURRENT_TIMESTAMP - '1 day'::interval
+        LIMIT 1
+    )
+    SELECT
```

Comment:
Now that this no longer has distinct on market ID, will the max time and first fill be correctly computed for the market ID when doing the later left join?